Tasks/KubernetesManifestV0/src/utils/utilities.ts

```diff
@@ -147,7 +148,21 @@ export function substituteImageNameInSpecFile(currentString: string, imageName:
     }, '');
 }
 
+export function getAPIVersion() {
+    if(apiVersion)
+        return apiVersion;
+
+    const kubectlPath = tl.which('kubectl', true)
```

Comment:
A Kubectl path should be floating around in the action, reuse that.

Reply:
Kubectl path has been used only once per action to extract the kubectl object. If I try to reuse that, a lot of function calls will have to be changed.